### ISSUE TITLE
Is your XPlaneCDU will also work with the default 737 FMS ?

### ISSUE THREAD

**Issue by yvesbbbb** [waynepiekarski/XPlaneCDU#6]
Hi Wayne,

Are there a lot of changes to the orginal code to make it work for the default FMS ? 

Or, is it a complete recoding ?

Thank you and have a nice day.

Yves

**Comment by yvesbbbb** [comment]
I ask the question because the Display DREFS for the default CDU
are now available.

https://developer.x-plane.com/article/datarefs-for-the-cdu-screen/

**Comment by waynepiekarski** [comment]
Yes, it is definitely on my list of tasks to implement for XPlaneCDU. I was talking with Philipp at FS Expo a few months ago about this, and asked them to put these datarefs in, so now I have to find time and do it.

**Comment by yvesbbbb** [comment]
Good, great news !!  I will be patient  ;-)
